debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/ObjectFieldInfo.java

```diff
@@ -37,6 +37,7 @@
 import com.ibm.j9ddr.vm29.pointer.helper.J9UTF8Helper;
 import com.ibm.j9ddr.vm29.pointer.helper.ValueTypeHelper;
 import com.ibm.j9ddr.vm29.structure.J9Object;
+import static com.ibm.j9ddr.vm29.structure.ObjectFieldInfo.OBJECT_SIZE_INCREMENT_IN_BYTES;
 import com.ibm.j9ddr.vm29.pointer.helper.J9ObjectHelper;
```

Comment:
Can you please organize these imports, putting `OBJECT_SIZE_INCREMENT_IN_BYTES` above with other static imports and removing the unused imports (`J9Object` and `J9ObjectHelper`).

Reply:
Sorry, it is `J9BuildFlags` and `J9Object` that are the unused imports: please remove them.

Reply:
Thanks.